runtime/inner-PEC.js

```diff
@@ -164,7 +164,7 @@ class InnerPEC {
         render(content) {
           this._pec._apiPort.Render({particle, slotName, content});
 
-          Object.keys(content).forEach(key => { this._requestedContentTypes.delete(key) });
+          Object.keys(content).forEach(key => { this._requestedContentTypes.delete(key); });
```

Comment:
Another arrow function that could use simplification (there are more below too).